<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[<br><a target="_blank" href="file:///C:\Users\Admin\eclipse-workspace\Automation-framework-bankguru/screenshotReportNG/TC_02_28_09_2021_12_51_45.png"><img src="file:///C:\Users\Admin\eclipse-workspace\Automation-framework-bankguru/screenshotReportNG/TC_02_28_09_2021_12_51_45.png" height='100' width='150'/> </a></br>]]>
    </line>
  </reporter-output>
  <suite started-at="2021-09-27T17:51:38Z" name="UploadFile" finished-at="2021-09-27T17:51:46Z" duration-ms="7960">
    <groups>
    </groups>
    <test started-at="2021-09-27T17:51:38Z" name="Run on FIREFOX" finished-at="2021-09-27T17:51:46Z" duration-ms="7960">
      <class name="takeScreenShot.screenshotTestFailed">
        <test-method is-config="true" signature="deleteAllFilesInReportNGScreenshot()[pri:0, instance:takeScreenShot.screenshotTestFailed@821330f]" started-at="2021-09-27T17:51:38Z" name="deleteAllFilesInReportNGScreenshot" finished-at="2021-09-27T17:51:38Z" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteAllFilesInReportNGScreenshot -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:takeScreenShot.screenshotTestFailed@821330f]" started-at="2021-09-27T17:51:38Z" name="beforeClass" finished-at="2021-09-27T17:51:45Z" duration-ms="6958" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="TC_02()[pri:0, instance:takeScreenShot.screenshotTestFailed@821330f]" started-at="2021-09-27T17:51:45Z" name="TC_02" finished-at="2021-09-27T17:51:45Z" duration-ms="50" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Google] but found [Google có các thứ tiếng: English Français 中文（繁體）]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Google] but found [Google có các thứ tiếng: English Français 中文（繁體）]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at takeScreenShot.screenshotTestFailed.TC_02(screenshotTestFailed.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<br><a target="_blank" href="file:///C:\Users\Admin\eclipse-workspace\Automation-framework-bankguru/screenshotReportNG/TC_02_28_09_2021_12_51_45.png"><img src="file:///C:\Users\Admin\eclipse-workspace\Automation-framework-bankguru/screenshotReportNG/TC_02_28_09_2021_12_51_45.png" height='100' width='150'/> </a></br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- TC_02 -->
        <test-method signature="Tc_01()[pri:0, instance:takeScreenShot.screenshotTestFailed@821330f]" started-at="2021-09-27T17:51:45Z" name="Tc_01" finished-at="2021-09-27T17:51:45Z" duration-ms="26" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tc_01 -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:takeScreenShot.screenshotTestFailed@821330f]" started-at="2021-09-27T17:51:45Z" name="afterClass" finished-at="2021-09-27T17:51:46Z" duration-ms="699" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- takeScreenShot.screenshotTestFailed -->
    </test> <!-- Run on FIREFOX -->
  </suite> <!-- UploadFile -->
</testng-results>
